sql-plugin/src/main/scala/org/apache/spark/sql/rapids/AggregateFunctions.scala

```diff
@@ -642,81 +643,77 @@ case class GpuSum(child: Expression,
       with GpuAggregateWindowFunction
```

Comment:
@abellina could you take a look at this file. Specifically around GpuSum and GpuAverage?  This is the only place that actually had any real merge conflicts.